mne/io/chpi.py

```diff
@@ -0,0 +1,259 @@
+# Authors: Eric Larson <larson.eric.d@gmail.com>
+#          Samu Taulu <staulu@uw.edu>
```

Comment:
cc @staulu once this is in a more complete state I'll probably bug you to take a look at the code, but you already have some credit :)